source/transactions/tests/error-labels.yml

```diff
@@ -959,3 +959,247 @@ tests:
       collection:
         data:
           - _id: 1
+
+  - description: do not add unknown commit label to MaxTimeMSExpired inside transactions
```

Comment:
In the transactions spec, you did add directions to add the UnknownTransactionCommitResult label if commitTransaction encounters a MaxTimeMSExpired error. Is the spec silent about errors encountered on other commands within a transaction, or am I just missing something that's not rendered within this PR diff?

Just want to confirm that we're not trying to test undefined behavior here.

I think the only other test in this file related to not adding labels is for DuplicateKeyException.

Reply:
This test ensures that drivers only add the "UnknownTransactionCommitResult" error label when `commitTransaction` fails with a maxTimeMS error and not when other commands fail with maxTimeMS errors.

>  Is the spec silent about errors encountered on other commands within a transaction, or am I just missing something that's not rendered within this PR diff?

Errors from other commands are discussed in the [TransientTransactionError section](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#transienttransactionerror). I don't think this PR needs to explicitly say that non-commitTransaction commands that fail with MaxTimeMSExpired must not be labelled.